src/components/forms/MemoryLimitSelector.tsx

```diff
@@ -4,13 +4,19 @@ import type { MemoryLimit } from "types/limits";
 import { BYTES_UNITS, MEM_LIMIT_TYPE } from "types/limits";
 import MemoryLimitAvailable from "components/forms/MemoryLimitAvailable";
 import { useCurrentProject } from "context/useCurrentProject";
+import type { InstanceAndProfileFormikProps } from "./instanceAndProfileFormValues";
 
 interface Props {
+  formik: InstanceAndProfileFormikProps;
```

Comment:
There is another use case in `ProjectResourceLimitsForm`, there we should report to total sum of memory available. Should this be a different mode of this component or wdyt?

<img width="1610" height="959" alt="image" src="https://github.com/user-attachments/assets/60151d11-93d1-4201-bf74-ae1abd3d7725" />